radar/web/src/map-info/cs_office/index.ts

```diff
@@ -19,12 +18,6 @@ export default {
     ],
 
     mapStyles: [
-        {
```

Comment:
Why do you want to remove the office simple radar version?

Reply:
Lol it was like 2am im adding it back